features/custom_matchers/define_matcher.feature

```diff
@@ -454,3 +454,34 @@ Feature: Define a custom matcher
       """
     When I run `rspec ./alias_spec.rb --format documentation`
     Then the exit status should be 0
+
+  Scenario: With expectation errors that bubble up
+
+    By default the match block will swallow expectation errors (e.g. caused by using an expectation such as `expect(1).to eq 2`), if you with to allow these to bubble up, pass in the option `:notify_expectation_failures => true`.
```

Comment:
Typo in "with". Also in [the source](https://github.com/rspec/rspec-expectations/blob/87588b1614a7be1c6560c5a024ccf8764fd125b9/lib/rspec/matchers/dsl.rb#L125) and [here](https://github.com/rspec/rspec-expectations/blob/87588b1614a7be1c6560c5a024ccf8764fd125b9/lib/rspec/matchers/dsl.rb#L154), too.

Reply:
Thank you, fixed in https://github.com/rspec/rspec-expectations/pull/1373/commits/ed6cf6804f577383993a6883da1a47e7192705a2.